### DISCUSSION TITLE
Clarification Regarding PrimeBlocks License Terms for Freelance Web Projects

### DISCUSSION THREAD

**Discussion by Ms-indi** [primefaces/community#1528] [Figma UI Kit]
Good afternoon, 

I am a freelancer and develop websites for Russian companies. 
Please confirm/clarify that, in accordance with the PrimeBlocks license terms, I can:
**first option:** for each project, purchase a license on your own behalf to use PrimeBlocks, and upon completion of the project (in order to transfer the final product with source files to the client) provide you with information about the client to link it to this license;
**second option:** develop projects within the framework of my license, upon completion of the project (in order to transfer the final product with source files to the client) I purchase a license for the client and provide you with information about the client to link this new license.
I also ask you to inform me in what order and what information about the end user - a Russian legal entity - I must provide you with.

Thank you in advance.

**Comment by onursenture** [comment]
Hi,

You can buy a license for each project and use PrimeBlocks to build it. Contact PrimeTek to transfer the license for more details: https://primetek.com.tr/#contact

You can use your license to develop projects. When finished, you can buy a license for your client for each project.

Remember, don't let clients use PrimeBlocks separately from the finished project, and don't share your license with others.
